lib/event_tracker/jobs/tracker_job.rb

```diff
@@ -0,0 +1,13 @@
+require 'active_job'
+
+module EventTracker
+  module Jobs
+    class TrackerJob < ::ActiveJob::Base
+      queue_as :default
+
+      def perform(doer_id, event_name, event_label, properties)
+        raise 'Implement in adapter'
```

Comment:
http://ruby-doc.org/core-2.3.0/NotImplementedError.html
```ruby
raise NotImplementedError.new('Implement in adapter')
```
could be a more accurate subclass of `Exception`